src/App.js

```diff
@@ -1,25 +1,107 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState, useEffect } from "react"
+import Menu from "./components/Menu"
+import Question from "./components/Question"
+import { nanoid } from "nanoid"
+import "./App.css"
+const App = () => {
```

Comment:
It seems like you have two functions named App(), why? I think you should use different names and be consistent when naming components, variables and functions.